From d55188596488c92394723003f2de4cbe7064e1ef Mon Sep 17 00:00:00 2001
From: Steinar Bakkemo <steinar.bakkemo@remarkable.no>
Date: Thu, 13 Sep 2018 13:21:21 +0200
Subject: [PATCH] avoid-autotests

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 108ec1e..7f557cb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -62,7 +62,7 @@ ecm_setup_version(PROJECT
     SOVERSION 5)
 
 add_subdirectory(src)
-add_subdirectory(autotests)
+#add_subdirectory(autotests)
 add_subdirectory(tests)
 
 
-- 
2.7.4

